src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessor.java

```diff
@@ -149,6 +167,18 @@ private void finaliseProcessedMessage(
         }
     }
 
+    private void deadLetterTheMessage(
+        IMessage message,
+        String reason,
+        String description
+    ) throws InterruptedException, ServiceBusException {
+        messageOperations.deadLetter(message.getLockToken(), reason, description);
+
+        log.info("Message with ID {} has been dead-lettered", message.getMessageId());
```

Comment:
should this not be an error?

Reply:
Was info before - the line has just been moved